src/components/Card/Card.tsx

```diff
@@ -2,41 +2,36 @@ import * as React from "react";
 import bem from "../../utils/bem";
 
 interface CardProps {
-    /** ID  */
-    id?: string;
-
+    /** Additional attributes passed to the card */
+    attributes?: {};
```

Comment:
Where are attributes used in `card` ?

Reply:
Whoops! Thanks for catching that—after adding in so many of these it's inevitable stuff would fall through the cracks :)